### ISSUE TITLE
Mobile navbar menu toggle not focusable with TAB sequences

### ISSUE THREAD

**Issue by fsologureng** [go-gitea/gitea#23053]
### Description

A person with screen reader over a mobile can't access to their own profile or to the Create button.

This can be checked with mobile emulation in a browser without a screen reader.

### Gitea Version

Codeberg

### Can you reproduce the bug on the Gitea demo site?

Yes

### Screenshots

![Capture with focus in previous tabindex element](https://user-images.githubusercontent.com/5055343/220488515-4f96e545-13c2-4b38-aa37-5b697e777b8a.png)

![Capture after pressing TAB](https://user-images.githubusercontent.com/5055343/220488622-5b988618-5bc4-44d9-82a3-ab2cd4271914.png)

### How are you running Gitea?

https://try.gitea.io

### Database

None

**Comment by HesterG** [comment]
I can see that the menu button on the right of the nav bar is not reachable by pressing tab. But i am not very sure which is the create button, is it "run the binary" link?

**Comment by wxiaoguang** [comment]
The requirement behind this issue is: click  the `Create` button to create new repositories.

![image](https://user-images.githubusercontent.com/2114189/220509360-7bfeed91-843c-4f1c-baa6-78dbf934893f.png)

**Comment by HesterG** [comment]
I think this might be a problem with accessibility. Right now the menu button is not accessible, leading to the unaccessiblity of buttons inside the menu like Create button.

**Comment by lunny** [comment]
Fixed by #23059
